includes/class-wc-product-variation.php

```diff
@@ -413,6 +416,34 @@ public function get_catalog_visibility( $context = 'view' ) {
 		return apply_filters( $this->get_hook_prefix() . 'catalog_visibility', $this->parent_data['catalog_visibility'], $this );
 	}
 
+	/**
+	 * Get attribute summary.
+	 *
+	 * By default, attribute summary contains comma-delimited 'attribute_name: attribute_value' pairs for all attributes.
+	 *
+	 * @param string $context What the value is for. Valid values are view and edit.
+	 *
+	 * @since 3.6.0
+	 * @return string
+	 */
+	public function get_attribute_summary( $context = 'view' ) {
+		$attribute_summary = $this->get_prop( 'attribute_summary', $context );
+		return apply_filters( $this->get_hook_prefix() . 'attribute_summary', $attribute_summary, $this );
```

Comment:
Why are you manually calling the filter here? `get_prop` handles filtering.

Reply:
You're right, thx. Got confused by other getters running their filters.